Xamarin.Forms.Platform.Android/Renderers/WebViewRenderer.cs

```diff
@@ -209,6 +217,26 @@ void UpdateMixedContentMode()
 			}
 		}
 
+		void UpdateEnableZoomControls()
+		{
+			// BuiltInZoomControls supported as of API level 3
+			if (Control != null && ((int)Build.VERSION.SdkInt >= 3))
```

Comment:
we do not support anything below API 19, so the check is not really required

Reply:
Removed redundant check